go/host/storage/hostdb/batch.go

```diff
@@ -41,13 +41,28 @@ func AddBatch(dbtx *dbTransaction, db HostDB, batch *common.ExtBatch) error {
 		return fmt.Errorf("could not encode L2 transactions: %w", err)
 	}
 
+	// this value is used on host side rollup size estimation
+	txDataSize := 0
+	if len(batch.TxHashes) > 0 {
+		// this will be compressed in the rollup, so apply estimated compression factor
+		txDataSize = len(batch.EncryptedTxBlob)
+	} else {
+		// empty batches still contribute to rollup size through:
```

Comment:
Wouldn't it be more intuitive to move this stuff out of th else, and then have:

```golang

...
if len(batch.TxHashes) > 0 {
		// this will be compressed in the rollup, so apply estimated compression factor
		txDataSize += len(batch.EncryptedTxBlob)
	} 
```

Reply:
Because the 20bytes is roughly how much the header compresses to.